PostHog/PostHogSDK.swift

```diff
@@ -493,6 +503,12 @@ let maxRetryDelay = 30.0
         return false
     }
 
+    private func setPersonPropertiesForFlagsIfNeeded(_ userProperties: [String: Any]?) {
```

Comment:
should we only allow doing this and group props if `personProfiles` is != `never`?